jme3-examples/src/main/java/jme3test/light/pbr/RefEnv.java

```diff
@@ -127,7 +127,7 @@ public void done(LightProbe result) {
                     rootNode.getChild(0).setCullHint(Spatial.CullHint.Dynamic);
                 }
             });
-            ((SphereProbeArea) probe.getArea()).setRadius(100);
+            probe.getArea().setRadius(100);
```

Comment:
I mean, the removal of the cast or not doesn't change that.  It was using the deprecated method before, too. I wonder what the alternative should be.

Reply:
We'd need to find nehon's comment on that, but basically something like if (probe.getArea() instanceof SphereProbeArea) etc.
Or here specifically keeping the cast, as getArea is not deprecated, but the assumption that it can only return a SphereProbeArea _is_
But we should double check that again with the reality.

Reply:
`LightProbe.getArea()` is not deprecated. Perhaps you were thinking of `LightProbe.getBounds()`?

Reply:
Oh yeah, I meant getBounds(), that was my source of confusion!
I wonder why we have a separate class "SphereProbeArea" though, for something that could be a float, when we don't plan to have multiple Area Types.